client/pdo/client/scripts/CreateCLI.py

```diff
@@ -90,6 +92,65 @@ def CreateContract(ledger_config, client_keys, enclaveclients, contract) :
 
     cclinit_result = initialize_response.submit_initialize_transaction(ledger_config)
 
+def are_the_urls_same(url1, url2):
+    # OK, we need a better way to compare URLS than the string comparison used here. 
+    # Ideally, one should normalize urls, and also resolve hostname to ip address before comparison
+
+    return (url1 == url2)
+
```

Comment:
There are canonical tools for doing this in urllib. Suggest we use them rather than re-invent the wheel. Or... why not just canonicalize URLs first then its just a string compare?

More of a question though, why are we using the URL as a key?